doc/source/dev/contributor/quickstart_mac.rst

```diff
@@ -0,0 +1,109 @@
+
+Installing SciPy
+--------------
+
+(Consider following along with the companion video `Anaconda SciPy Dev: Part II (macOS)`_)
+
+Currently we have *two* versions of SciPy: the latest release as installed by Anaconda, and the development version we just built. Ideally, we'd like to be able to switch between the two as needed. `Virtual environments <https://medium.freecodecamp.org/why-you-need-python-environments-and-how-to-manage-them-with-conda-85f155f4353c>`_ can do just that. With a few keystrokes in the terminal or even the click of an icon, we can enable or disable our development version. Let's set that up.
+
+#. In a terminal window, enter ``conda list``. |br| This shows a list of all the Python packages that came with the Anaconda distribution of Python.
+
+#. Enter ``conda create --name scipydev``. |br| This tells conda to creates a virtual environment named ``scipydev``. Note that `scipydev` can be replaced by any name you'd like to refer to your virtual environment.
+
+#. You're still in the base environment. Activate your new virtual environment by entering ``conda activate scipydev``. |br| If you're working with an old version of ``conda``, you might need to type ``source activate scipydev`` instead (see `here <https://stackoverflow.com/questions/49600611/python-anaconda-should-i-use-conda-activate-or-source-activate-in-linux)>`_.
+
+#. (Optional) Enter ``conda list`` again. Note that the new virtual environment has no modules installed. If you were to open a Python interpreter now, you wouldn't be able to import ``numpy``, ``scipy``, etc...
+
+#. Enter ``conda install cython numpy matplotlib pytest spyder``. |br| Note that we're only installing SciPy's dependencies (and Spyder so we can use the IDE), but not SciPy itself.
+
+#. Our goal now is to add our root SciPy directory to the ``PYTHONPATH`` environment variable whenever this virtual environment is activated. This will ensure that Python can find the SciPy code we are trying to `import`. This requires adding a few files and folders deep inside your Anaconda installation directory. I suggest watching `the video <https://youtu.be/Faz29u5xIZc?t=35>`_ for this part. To summarize, you want to create:
+
+   - ``/anaconda3/envs/scipydev/conda/activate.d/env_vars.sh``, and
+
+   - ``/anaconda3/envs/scipydev/conda/deactivate.d/env_vars.sh`` |br| where: |br|
+
+   - ``anaconda3`` is the root directory of your Anaconda installation;
+
+   - ``conda``, ``activate.d``, and ``deactivate.d`` are new folders; and
+
+   - ``env_vars.sh`` is the name of two new plain text files |br| with the contents |br|
+   
+   - ``export PYTHONPATH=/scipy`` (where ``scipy`` is to be replaced with the full path of the SciPy root directory), and
+   
+   - ``unset PYTHONPATH``,
+   
+   respectively.
```

Comment:
At first, it seemed a bit unclear that the ~/activate.d/env_vars.sh file needed the "export PYTHONPATH=/scipy" line, and that the ~/deactivate.d/env_vars.sh file needed the "unset PYTHONPATH" line. Originally, I put both lines into both files, and only remedied this after viewing the videos. Making it more clear ahead of time (as opposed to adding "respectively" after the fact) which line goes in which file would be helpful.

Reply:
Noted.

Reply:
I don't believe a fix for this is necessary, if we use the conda develop instead.

Reply:
OK. The thing is that I can't easily fix the video. I was thinking maybe offering both options?

Reply:
Makes sense. I'll rewrite it and offer both options.